h2o-r/h2o-package/R/models.R

```diff
@@ -567,10 +567,10 @@ setMethod("h2o.transform", signature("H2OTargetEncoderModel"), function(model, d
 #'
 #' # Transform words to vectors without aggregation
 #' sentences <- as.character(as.h2o(c("b", "c", "a", NA, "b")))
-#' h2o.transform(w2v_model, sentences) # -> 5 rows total, 2 rows NA ("c" is not in the vocabulary)
+#' h2o.transform_word2vec(w2v_model, sentences) # -> 5 rows total, 2 rows NA ("c" is not in the vocabulary)
```

Comment:
The method defines below is `h2o.transform` not `h2o.transform_word2vec`. That's why the tests fail I guess.

I don't understand why we need such an example above `h2o.transform` method. This should probably be in `h2o.transform_word2vec` in `w2vutils.R` file. I can make the change if you agree.

Reply:
Sure! You're always welcome to make changes. 
And the goal is to include examples for each R function. I guess I'm still confused about transform vs. transform_word2vec :flushed: